ckan/lib/webassets_tools.py

```diff
@@ -0,0 +1,139 @@
+# encoding: utf-8
+
+import logging
+import os
+
+from markupsafe import Markup
+
+from webassets import Environment
+from webassets.loaders import YAMLLoader
+
+from ckan.common import config, g
+
+logger = logging.getLogger(__name__)
+env = None
+
+
+def create_library(name, path):
+    """Create WebAssets library(set of Bundles).
+    """
+    config_path = os.path.join(path, u'webassets.yml')
+    if not os.path.exists(config_path):
+        return
+
+    library = YAMLLoader(config_path).load_bundles()
+    bundles = {
+        u'/'.join([name, key]): bundle
+        for key, bundle
+        in library.items()
+    }
+
+    # Unfortunately, you'll get an error attempting to register bundle
+    # with the same name twice. For now, let's just pop existing
+    # bundle and avoid name-conflicts
+    # TODO: make PR into webassets with preferable solution
```

Comment:
can we link to an open issue on the webassets repo?

Reply:
Done